internal/install/application_configuration.go

```diff
@@ -153,13 +154,13 @@ func (ac *ApplicationConfiguration) SetCurrentProfile(name string) {
 // This is mandatory as "elastic-agent-complete" is available since 7.15.0-SNAPSHOT.
 func selectElasticAgentImageName(version, agentBaseImage string) string {
 	if version == "" { // as version is optional and can be empty
-		return elasticAgentLegacyImageName
+		return elasticAgentWolfiImageName
```

Comment:
WDYT about changing these defaults values to use the wolfi image docker.elastic.co/elastic-agent/elastic-agent-wolfi (e.g. no version specified) instead of the legacy image (beats namespace) docker.elastic.co/beats/elastic-agent?

Reply:
We can adapt this to the new default, yes, if we are sure that in old versions the beats namespace is used.

Reply:
Yes, the same behaviour is kept.
As a reference, I add here the docker images used in different versions:
- 7.14.0
  ```
   $ docker ps --format "{{.Names}} {{.Image}}" |grep elastic-agent
  elastic-package-stack-elastic-agent-1 docker.elastic.co/beats/elastic-agent:7.14.0
  elastic-package-stack-fleet-server-1 docker.elastic.co/beats/elastic-agent:7.14.0
  ```
- 7.15.0
  ```
   $ docker ps --format "{{.Names}} {{.Image}}" |grep elastic-agent
  elastic-package-stack-elastic-agent-1 docker.elastic.co/beats/elastic-agent-complete:7.15.0
  elastic-package-stack-fleet-server-1 docker.elastic.co/beats/elastic-agent-complete:7.15.0
  ```
- 8.2.0
  ```
   $ docker ps --format "{{.Names}} {{.Image}}" |grep elastic-agent
  elastic-package-stack-elastic-agent-1 docker.elastic.co/elastic-agent/elastic-agent-complete:8.2.0
  elastic-package-stack-fleet-server-1 docker.elastic.co/elastic-agent/elastic-agent-complete:8.2.0
  ```
- 8.16.0 (SNAPSHOT)
  ```
   $ docker ps --format "{{.Names}} {{.Image}}" |grep elastic-agent
  elastic-package-stack-elastic-agent-1 docker.elastic.co/elastic-agent/elastic-agent-wolfi:8.16.0-SNAPSHOT
  elastic-package-stack-fleet-server-1 docker.elastic.co/elastic-agent/elastic-agent-wolfi:8.16.0-SNAPSHOT
  ```
- 9.0.0 (SNAPSHOT)
  ```
   $ docker ps --format "{{.Names}} {{.Image}}" |grep elastic-agent
  elastic-package-stack-elastic-agent-1 docker.elastic.co/elastic-agent/elastic-agent-wolfi:9.0.0-SNAPSHOT
  elastic-package-stack-fleet-server-1 docker.elastic.co/elastic-agent/elastic-agent-wolfi:9.0.0-SNAPSHOT
  ```